server/src/models/Task.ts

```diff
@@ -0,0 +1,13 @@
+class Task {
+  private timeStart: number;
+
+  constructor(public task: string) {
+    this.task = task;
```

Comment:
Staraj sie tak nie robic, nie ma to zadnego sensu ze na Tasku jest task. Zamiast tego staraj sie nazywac rzeczy po imieniu ( wiem ze to trudne w programowaniu :D) ale tutaj powinno byc np.: task.title lub task.text